<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUserDialog</class>
 <widget class="QDialog" name="AddUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="confirmPushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>75</width>
     <height>83</height>
    </rect>
   </property>
   <property name="text">
    <string>用户班级：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="classNumberLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>171</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>75</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string>用户密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>171</width>
     <height>66</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>11</y>
     <width>77</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>用户类型：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>用户ID：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>11</y>
     <width>171</width>
     <height>211</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QComboBox" name="comboBox">
    <item>
     <property name="text">
      <string>老师</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>学生</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="idLineEdit"/>
   <widget class="QLineEdit" name="lineEdit"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
